trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/DummyGravitinoConnector.java

```diff
@@ -20,8 +30,52 @@
  */
 public class DummyGravitinoConnector implements Connector {
 
-  public DummyGravitinoConnector() {
+  private final String metalake;
+  private final CatalogConnectorManager catalogConnectorManager;
+
+  private final Set<Procedure> procedures = new HashSet<>();
+
+  public DummyGravitinoConnector(String metalake, CatalogConnectorManager catalogConnectorManager) {
     super();
+    this.metalake = metalake;
+    this.catalogConnectorManager = catalogConnectorManager;
+
+    try {
+      // call gravitino.system.createCatalog(metalake, catalog, provider, properties, ignoreExist)
+      MethodHandle createCatalog =
+          MethodHandles.lookup()
+              .unreflect(
+                  DummyGravitinoConnector.class.getMethod(
+                      "createCatalog", String.class, String.class, String.class, boolean.class))
+              .bindTo(this);
+
+      List<Procedure.Argument> arguments =
+          List.of(
+              new Procedure.Argument("CATALOG", VARCHAR),
+              new Procedure.Argument("PROVIDER", VARCHAR),
+              new Procedure.Argument("PROPERTIES", VARCHAR, false, "{}"),
+              new Procedure.Argument("IGNORE EXIST", BOOLEAN, false, true));
+      Procedure procedure = new Procedure("system", "createCatalog", arguments, createCatalog);
+      procedures.add(procedure);
+
+      // call gravitino.system.dropCatalog(metalake, catalog, ignoreNotExist)
+      MethodHandle dropCatalog =
+          MethodHandles.lookup()
+              .unreflect(
+                  DummyGravitinoConnector.class.getMethod(
+                      "dropCatalog", String.class, boolean.class))
+              .bindTo(this);
+      arguments =
+          List.of(
+              new Procedure.Argument("CATALOG", VARCHAR),
+              new Procedure.Argument("IGNORE NOT EXIST", BOOLEAN, false, true));
+      procedure = new Procedure("system", "dropCatalog", arguments, dropCatalog);
```

Comment:
Is trino case insensitive, what will be happened if user write as "DROPCATALOG"? I would suggest to use "drop_catalog", "create_catalog".

Reply:
Ok